### PR TITLE
feat: Add revoke vc to demo

### LINKED ISSUE #102: Add  revoke vc to demo

### PR DESCRIPTION
**Pull request by fqutishat**
closes #102

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: pkg/restapi/issuer/operation/operations.go [hunk 1]
```diff
@@ -30,8 +30,11 @@ const (
 	login    = "/login"
 	callback = "/callback"
 	retrieve = "/retrieve"
+	revokeVC = "/revokeVC"
```
Comment (sandrask) [inline pkg/restapi/issuer/operation/operations.go]: it should be either revoke or retrieveVC; probably revoke

File: pkg/restapi/issuer/operation/operations.go [hunk 2]
```diff
@@ -212,6 +219,58 @@ func (c *Operation) RetrieveVC(w http.ResponseWriter, r *http.Request) {
...
+			fmt.Sprintf("failed to parse form: %s", err.Error()))
+
+		return
+	}
+
+	reqBytes, err := prepareUpdateCredentialStatusRequest(r.Form.Get("vcDataInput"),
+		"Revoked", "Disciplinary action")
+	if err != nil {
+		c.writeErrorResponse(w, http.StatusInternalServerError,
+			fmt.Sprintf("failed to perpare update credential status request: %s", err.Error()))
```
Comment (sandrask) [inline pkg/restapi/issuer/operation/operations.go]: to prepare

File: pkg/restapi/issuer/operation/operations.go [hunk 3]
```diff
@@ -395,6 +464,7 @@ func (c *Operation) registerHandler() {
 		support.NewHTTPHandler(login, http.MethodGet, c.Login),
 		support.NewHTTPHandler(callback, http.MethodGet, c.Callback),
 		support.NewHTTPHandler(retrieve, http.MethodGet, c.RetrieveVC),
+		support.NewHTTPHandler(revokeVC, http.MethodPost, c.revokeVC),
```
Comment (sandrask) [inline pkg/restapi/issuer/operation/operations.go]: lowercase?

File: pkg/restapi/issuer/operation/operations.go (hunk 1 above)
Comment (fqutishat) [reply pkg/restapi/issuer/operation/operations.go]: done

File: pkg/restapi/issuer/operation/operations.go (hunk 2 above)
Comment (fqutishat) [reply pkg/restapi/issuer/operation/operations.go]: done

File: pkg/restapi/issuer/operation/operations.go (hunk 3 above)
Comment (fqutishat) [reply pkg/restapi/issuer/operation/operations.go]: done

**Pull request merged**